packages/gatsby-source-shopify/README.md

```diff
@@ -72,6 +72,13 @@ plugins: [
       // Possible values are: 'shop' and 'content'.
       // Defaults to ['shop', 'content'].
       includeCollections: ["shop", "content"],
+      // Download Images Locally
+      // set to false if you plan on using shopify's CDN
+      downloadImages: true,
+      // default image URL
+      // if you set downloadImages to false then it will set all localFile of file from this URL this is to make graphQL types unchanged
+      defaultImageURL:
+        "https://raw.githubusercontent.com/mrhut10/gatsby/Feature/gatsby-source-shopify/downloadImagesOption/packages/gatsby-source-shopify/src/default.png",
```

Comment:
why is this necessary? If you're not downloading images then we wouldn't want `localFile` on the schema correct cause there's nothing to query there?

Reply:
yeah I was swapping all of the images with a single placeholder image as wasn't sure if I would break something if a image wasn't supplied, but as per your comment on line `packages/gatsby-source-shopify/src/nodes.js:41` sounds like much better solution just to skip over a few lines with a nested if statement :)

Reply:
sweet fixed this up,
you can see when downloadImages is set to false, true (default) then the localFile on the image node will or wont be in the graphQl types.

![image](https://user-images.githubusercontent.com/20330237/104127861-9b541980-53b8-11eb-8848-3e111182cddb.png)